bedrock/base/templates/macros.html

```diff
@@ -188,7 +187,13 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     {% do fxa_queries.append('entrypoint_variation=' ~ entrypoint_variation) %}
   {% endif %}
 
-  href="{{ settings.FXA_ENDPOINT }}{{ fxa_path }}?service={{ service_type }}&{{ fxa_queries|join('&') }}" data-mozillaonline-link="{{ settings.FXA_ENDPOINT_MOZILLAONLINE }}{{ fxa_path }}?service={{ service_type }}&{{ fxa_queries|join('&') }}"
+  {# service_type is option for Firefox 71+ (issue #8198) #}
+  {% if service_type %}
+    {% do fxa_queries.insert(0, 'service=' ~ service_type) %}
+    {% do fxa_queries.append('context=fx_desktop_v3') %}
```

Comment:
I added `context=fx_desktop_v3` to this block because it is required if `service=sync`.
https://mozilla.github.io/application-services/docs/accounts/metrics.html#descriptions-of-metrics-related-query-parameters

Reply:
Would it be slightly more readable to insert `context=fx_desktop_v3` right after `service` in the href, seeing as the two are interrelated? Right now, one appears at the start of the params and the other at the end.